src/component/ChairList/ChairList.js

```diff
@@ -4,29 +4,31 @@ import styled from 'styled-components';
 
 import ChairCard from '../ChairCard/ChairCard';
 
-export default function ChairList({chairsData, toggleDescription, showDescription}) {
+export default function ChairList({chairs, toggleDescription, showDescription, setFavoriteChair, toggleLike}) {
   const {name} = useParams();
   let navigate = useNavigate();
 
+  const filteredChairs = name ? chairs.filter(chair => name === chair.style) : chairs;
+
   return (
     <ChairContainer>
       <HeadingContainer>
         <button onClick={() => navigate('/')}>
           <StyledBackButtonIcon />
         </button>
-        <h1>{name} chairs</h1>
+        {name ? <h1>{name} chairs</h1> : <h1>Favorite Chairs</h1>}
```

Comment:
This would be a shorter way of writing it. Should basically do the same thing, both are correct though and I just wanted to show another way of writing it without a lot of repetition :)
```suggestion
        <h1>{name ? `${name}` : 'Favorite'} Chairs</h1>
```